tests/sys_crypto/tests-crypto-poly1305.c

```diff
@@ -19,198 +19,299 @@ static const uint8_t msg_1[64] = { 0 };
 static const uint8_t tag_1[16] = { 0 };
 
 static const uint8_t key_2[32] = {
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-    0x36, 0xe5, 0xf6, 0xb5, 0xc5, 0xe0, 0x60, 0x70, 0xf0, 0xef, 0xca, 0x96, 0x22, 0x7a, 0x86, 0x3e
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
For all keys I would either leave them as they were before or key 8 values per row. But I would definitely not keep the result of uncrustify.

Reply:
Reverted the file 69f79847

Reply:
There was no need to revert the whole, just the changes I pointed out.